### PR TITLE
fix(supernode/server): add error handling

### PR DESCRIPTION
**Pull request by hwdef**
### Ⅰ. Describe what this PR did

add error handling in test and version folder

### Ⅱ. Does this pull request fix one issue?

Ref: #952

### CODE REVIEW

**Comment by Starnop** [comment]
Could you please open an issue to make a plan and track the progress for `err handling`?

WDYT?

**Comment by hwdef** [comment]
Thanks for your suggestion.
I will do it .

File: version/version.go [hunk 1]
```diff
@@ -125,7 +125,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Write(data)
+		if _, err := w.Write(data); err != nil {
+			http.Error(w, fmt.Sprintf("error writing data: %s", err), http.StatusInternalServerError)
```
Comment (Starnop) [inline version/version.go]: In fact, http.Error will call the function of `w.Write`. And that's meaningless. So no need to handle the error here.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**